modules/account-lib/package.json

```diff
@@ -33,7 +33,7 @@
   "license": "ISC",
   "dependencies": {
     "@bitgo/bls": "^0.1.0",
-    "@bitgo/statics": "^6.0.0",
+    "@bitgo/statics": "^6.1.0-rc.1",
```

Comment:
Please don't modify the versions of packages which are inside this monorepo (that's `@bitgo/statics`, `bitgo` or `@bitgo/account-lib` for now). These versions are managed by the monorepo management tool lerna during release, and modifying it outside a release branch will create merge conflicts when I release new versions.

Reply:
Hi  @tylerlevine.

We need statics version 6.1.0 not only to run the tests but also to being able to import casper-client as dependency.
How should we proceed with this if we cannot upgrade this lib until release?

Reply:
When you are developing in this monorepo, the dependencies for packages within the monorepo are not downloaded and installed from npm. Instead they are symlinked to the version in the monorepo. If you have already made the changes you need in `statics`, then there is no need to update this version and you will get the version that currently exists in `master`.